community-operators/jaeger/jaeger.v1.8.2.clusterserviceversion.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     containerImage: docker.io/jaegertracing/jaeger-operator:1.8.2
     createdAt:  2019-01-09T12:00:00Z
     support: Jaeger
+    alm-examples: >-
+      [ { "apiVersion": "io.jaegertracing/v1alpha1", "kind": "Jaeger", "metadata": { "name": "agent-as-daemonset" }, "spec":{ "agent": { "strategy": "DaemonSet", "options":{ "log-level": "debug" } } } }, { "apiVersion": "io.jaegertracing/v1alpha1", "kind": "Jaeger", "metadata":{ "name": "simplest" } } ]
```

Comment:
Question: Does OpenShift console show these examples?

Reply:
Console uses the `alm-examples` annotation to populate the YAML editor.